src/Sylius/Bundle/ApiBundle/Command/AddProductReview.php

```diff
@@ -50,13 +50,13 @@ public function __construct(
         string $title,
         int $rating,
         string $comment,
-        string $productCode,
+        string $product,
```

Comment:
This is definitely something we should investigate more about - currently we need to consider the constructor's variable names as a part of BC promise.